### ISSUE TITLE
Windows mount drive on first start

### ISSUE THREAD

**Issue by bitlogik** [Scille/parsec-cloud#1301]
After each installation of v0.14 in Windows, on the first start, the mount drive is not mounted. Even the workspace slider in the GUI does nothing about this. Parsec needs a quit and restart to mount the drive.
Everything is fine in the GUI, but no mountpoint is visible in Windows. When clicking on "open the workspace in the file explorer" opens a explorer window with an error "not found".

The log doesn't tell about anything except :
```
{"task_name": "parsec.core.gui.trio_thread.QtToTrioJob._run_fn", "total_time": 1.016, "cpu_time": 0.141, "io_time": 0.875, "stack_before": "", "stack_after": "
File \"C:\\Program Files (x86)\\Parsec\\site-packages\\parsec\\service_nursery.py\", line 181, in wrap_child
    await async_fn(*args, task_status=task_status)
    File \"C:\\Program Files (x86)\\Parsec\\site-packages\\parsec\\core\\gui\\trio_thread.py\", line 94, in _run_fn
    result = await self._fn(*self._args, **self._kwargs)
    File \"C:\\Program Files (x86)\\Parsec\\site-packages\\parsec\\core\\gui\\new_version.py\", line 42, in _do_check_new_version
    latest_from_head, json_releases = await trio.to_thread.run_sync(_fetch_json_releases)
",
"event": "Blocking IO detected", "logger": "parsec.monitoring", "level": "warning", "timestamp": "2020-07-23 22:50:29"}
```
seems not related

The main goal of this post issue is to know whether other testers are facing the same issue.

**Comment by vxgmichel** [comment]
Hi @bitlogik,

Thanks a lot for the report, it's something we noticed two after the pre-release yesterday. It's a very annoying issue since it's hard to reproduce and it doesn't report any error on the application side. Here's the info I've been able to gather so far:

- it **does** happen when upgrading to parsec `1.14`
- it **does** keep happening after enabling/disabling workspace from the application
- it **does not** keep happening after a restart of the application
- it **does not** happen on a fresh install
- it **does not** happen after a downgrading and upgrading back to `1.14` (e.g. `1.14 -> 1.13 -> 1.14`)

Please let us know if you're able to confirm or correct any of this information !

**Comment by vxgmichel** [comment]
We figured it out, it was related to some privilege issues when running parsec with the windows installer. It'll be fixed soon.

**Comment by vxgmichel** [comment]
Fixed in #1303
